_includes/footer.html

```diff
@@ -3,21 +3,27 @@
     <div>
       <img src="/assets/images/logo-gsa.png" alt="GSA logo">
     </div>
-    <div itemscope itemtype="http://schema.org/ContactPoint">
-      <h1>Contact Us</h1>
-      <a href="mailto:18f@gsa.gov" itemprop="email"><i class="fa fa-envelope"></i> 18F@gsa.gov</a>
+    <div>
+      <h1>Work</h1>
+      <a href="/consulting">Consulting</a>
+      </br>
+      <a href="/delivery">Delivery</a>
+      </br>
+      <a href="/dashboard">Dashboard</a>
       <br/>
-      <a href="https://twitter.com/18F"><i class="fa fa-twitter"></i> @18F</a>
+      <a href="https://github.com/18F"><i class="fa fa-github"></i> github.com/18F</a>
     </div>
     <div>
-      <h1>Fork Our Code</h1>
-      <a href="https://github.com/18F"><i class="fa fa-github"></i> github.com/18F</a>
-      <br/>
-      <a href="https://github.com/presidential-innovation-fellows"><i class="fa fa-github"></i> github.com/presidential-innovation-fellows</a>
-      <br/>
-      <a href="https://github.com/18F/18f.gsa.gov/edit/staging/{{ page.path }}">Edit this page</a>
-      <br/>
-      <a href="/developer/">Developer resources</a>
+      <h1> People </h1>
+      <a href="/team">Team</a>
+    </br>
+      <a href="https://pages.18f.gov/joining-18f/">Join us</a>
+    </div>
+    <div>
```

Comment:
Another idea I had would be to use <section> html5 tags for these. Not strictly necessary, but may make more semantic sense.